WordPressSharedTests/WPAnalyticsTests.swift

```diff
@@ -0,0 +1,133 @@
+import Quick
+
+@testable import WordPressShared
+
+class WPAnalyticsMethodBehaviorExamplesConfiguration: QuickConfiguration {
+    override class func configure(_ configuration: QCKConfiguration!) {
+        sharedExamples("a WPAnalyticsTracker method") { context in
+            guard let context else {
+                XCTFail("Test case needs to provide a context")
+                return
```

Comment:
Side note: Once can also inline this, like `return XCTFail(...)`.

It's probably not a good idea in this test suite because it's not a pattern it uses but I find it neat, specially under the hood. At first look it looks weird but thinking about it it's obviously okay. `XCTFail` returns `Void` so it's totally legit to "return it" because it's the same as `return Void` or  `return`.

🤓